backend/src/main/java/com/openelements/maven/initializer/backend/service/ProjectGeneratorService.java

```diff
@@ -119,16 +127,20 @@ public byte[] createProjectZip(String projectPath) {
 
     try {
       ByteArrayOutputStream baos = new ByteArrayOutputStream();
-      try (ZipOutputStream zos = new ZipOutputStream(baos)) {
+      try (ZipArchiveOutputStream zos = new ZipArchiveOutputStream(baos)) {
```

Comment:
Starting at line 128, I would refactor the try-catch-clock to a separate method.